### DISCUSSION TITLE
I would like to ask about the temperature-based valve opening adjustment (PID control) method.

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10658] [Q&A] [answered]
I want the PID control to work when AirTemp is below 30℃ and to reduce the valve opening to 50%.

1. Does the attached configuration look correct?
2. Which parameter of valveLinear should be set to reduce the valve opening to 50% when the temperature is below 30℃?

Please advise.

![image](https://user-images.githubusercontent.com/131832405/236710835-f7231d38-2e0f-4b09-b3c9-9cf2990de8af.png)

**Comment by casella** [comment, marked as answer]
> I want the PID control to work when AirTemp is below 30℃ and to reduce the valve opening to 50%.
> 
>     1. Does the attached configuration look correct?

Looks reasonable.

>     2. Which parameter of valveLinear should be set to reduce the valve opening to 50% when the temperature is below 30℃?

No such feature. You need to implement a control logic to do that.
> 
> 
> Please advise.
> 
> ![image](https://user-images.githubusercontent.com/131832405/236710835-f7231d38-2e0f-4b09-b3c9-9cf2990de8af.png)

**Reply by ZackeyZAKI** [reply]
Thank you for providing an answer.
